carrot-core/src/test/java/org/bigbase/carrot/CarrotCoreBase.java

```diff
@@ -0,0 +1,57 @@
+package org.bigbase.carrot;
+
+import org.bigbase.carrot.compression.Codec;
+import org.bigbase.carrot.compression.CodecFactory;
+import org.bigbase.carrot.compression.CodecType;
+import org.junit.AfterClass;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Objects;
+
+@RunWith(Parameterized.class)
+public class CarrotCoreBase {
+
+  protected static Codec codec;
+
+  // static {
+  //  UnsafeAccess.debug = true;
+  // }
+
+  @AfterClass
+  public static void cleanAll() throws IOException {
+    // Tear down previous run if it was
+    tearDown();
+  }
+
+  public CarrotCoreBase(Object c) throws IOException {
+    codec = (Codec) c;
+    // Tear down previous run if it was
+    tearDown();

```

Comment:
remove tearDown and setUp from base class

Reply:
Done